server/dash/dashboards/neighborhood.py

```diff
@@ -37,63 +39,90 @@ def populate_options():
+        Plotly graph that compares the selected moving average between the
+          chosen council and the moving average of the avg of all the 99 neighborhood councils.
     """
+    NC_POP = int(
+        pop[pop.council_name == selected_council].sum_pop20
+    )  # population of selected council.
+
+    NC_MOVING_AVERAGE_KEY = (
+        "nc_ma"  # Key for neighborhood council moving average in dataframe (total/99).
+    )
+    NC_POP_MOVING_AVERAGE_KEY = "nc_ma_pop"  # Key for neighborhood council moving average adjusted to per 10,000 people.
+    SELECT_NC_MOVING_AVERAGE_KEY = "select_nc_ma"  # Key for selected neighborhood council moving average in dataframe.
+    SELECT_NC_POP_MOVING_AVERAGE_KEY = "select_nc_ma_pop"  # Key for selected neighborhood council moving average in dataframe adjusted to per 10,000 people.
+
     neighborhood_sum_df = (
         df[df.council_name == selected_council]
         .groupby(["created_date"])
         .agg("sum")
         .reset_index()
     )  # noqa
+
     total_sum_df = df.groupby(["created_date"]).agg("sum").reset_index()
 
-    total_sum_df["nc_ma"] = (
+    total_sum_df[NC_MOVING_AVERAGE_KEY] = (
         total_sum_df.counts.rolling(selected_timeframe, center=True).mean()
         / NEIGHBORHOOD_COUNCILS_NUMBER
     )
-    neighborhood_sum_df["select_nc_ma"] = neighborhood_sum_df.counts.rolling(
-        selected_timeframe, center=True
-    ).mean()
-
+    total_sum_df[NC_POP_MOVING_AVERAGE_KEY] = (
+        total_sum_df.counts.rolling(selected_timeframe, center=True).mean()
+        / TOT_POP
+        * 10000
+    )
+    neighborhood_sum_df[
+        SELECT_NC_MOVING_AVERAGE_KEY
+    ] = neighborhood_sum_df.counts.rolling(selected_timeframe, center=True).mean()
+    neighborhood_sum_df[SELECT_NC_POP_MOVING_AVERAGE_KEY] = (
+        neighborhood_sum_df.select_nc_ma / NC_POP * 10000
+    )
     merged_df = pd.merge(
         neighborhood_sum_df, total_sum_df, on=["created_date", "created_date"]
     )
 
     fig = px.line(
         merged_df,
         x="created_date",
-        y=["select_nc_ma", "nc_ma"],
+        y=[
+            SELECT_NC_MOVING_AVERAGE_KEY,
+            SELECT_NC_POP_MOVING_AVERAGE_KEY,
+            NC_MOVING_AVERAGE_KEY,
+            NC_POP_MOVING_AVERAGE_KEY,
+        ],
         color_discrete_sequence=DISCRETE_COLORS,
-        labels={"created_date": "311 Request Date", "value": "Total Requests"},
-        title=selected_council + " vs 99 Neighborhood Councils 311 Requests Average",
+        labels={"created_date": "Request Date", "value": "Total Requests"},
+        title=selected_council + " vs Neighborhood Councils Average",
     )
-    newnames = {
-        "select_nc_ma": selected_council,
-        "nc_ma": "99 Neighborhood Councils Average",
+    new_names = {
+        SELECT_NC_MOVING_AVERAGE_KEY: selected_council,
+        NC_MOVING_AVERAGE_KEY: "Neighborhood Councils Average",
+        SELECT_NC_POP_MOVING_AVERAGE_KEY: selected_council + "council adjusted",
```

Comment:
Let's make it clear to the user that we're calculating the number of 311 requests per 10,000 people.

Reply:
Changed the title of the graph